docs/content/development/extensions-core/test-stats.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: doc_page
+---
+
+# Test Stats Aggregators
+
+Incorporates test statistics related aggregators, including z-score and p-value. Please refer to [https://www.paypal-engineering.com/2017/06/29/democratizing-experimentation-data-for-product-innovations/](https://www.paypal-engineering.com/2017/06/29/democratizing-experimentation-data-for-product-innovations/) for background and details.
+
+Make sure to include `druid-stats` extension in order to use these aggregrators.
+
+## Z-Score for two sample ztests post aggregator
+
+Please refer to [https://www.isixsigma.com/tools-templates/hypothesis-testing/making-sense-two-proportions-test/](https://www.isixsigma.com/tools-templates/hypothesis-testing/making-sense-two-proportions-test/) and [http://www.ucs.louisiana.edu/~jcb0773/Berry_statbook/Berry_statbook_chpt6.pdf](http://www.ucs.louisiana.edu/~jcb0773/Berry_statbook/Berry_statbook_chpt6.pdf) for more details.
+
+z = (p1 - p2) / S.E.  (assuming null hypothesis is true)
+
+where S.E. stands for standard error, and
+
+S.E. = sqrt{ p1 * ( 1 - p1 )/n1 + p2 * (1 - p2)/n2) }
+
+(p1 – p2) is the observed difference between two sample proportions.
+
+### zscore2sample post aggregator
+* **`zscore2sample`**: calculate the z-score using two-sample z-test while converting binary variables (***e.g.*** success or not) to continuous variables (***e.g.*** conversion rate).
+
+```json
+{
+  "type": "zscore2sample",
+  "name": "<output_name>",
+  "fields": [<count 1 (post_aggregator1)>, <sample size 1 (post_aggregator2)>, <count 2 (post_aggregator3)>, <sample size 2 (post_aggregator4)>]
```

Comment:
> However, I don't think it is more error prone or less intuitive, than using individual arguments.

With the current design, users should remember the meaning of arguments according to their appearance order in the fields list or see the document every time. What's worse, users are difficult to figure out even though they input the arguments in wrong order (e.g., ```[<count 2 (post_aggregator3)>, <sample size 2 (post_aggregator4)>, <count 1 (post_aggregator1)>, <sample size 1 (post_aggregator2)>]```).

If they should be specified individually, users, at least, cannot input arguments of wrong number or in wrong order.

> It is aligned and consistent with the way fields are used in Druid.

In this case, I think the usability (stated above) matters rather than consistency. There is no point for arbitrary number of arguments.

> As we might fine-tune, or evolve the input parameters so that the stat functions can be most useful and powerful, keep them in fields at this juncture should be good, so it can be continuously evolved / fine-tuned without dramatically changing the input signature.

If you have any plan, please let us know. I guess even changing input signature is not a big problem because arguments can be easily added and removed (thanks to json).